cogs/autothread.py

```diff
@@ -1,19 +1,20 @@
 from nextcord.ext import commands
+from nextcord import Message
 
-AUTO_THREAD_CHANNEL = 937840096021979236
+AUTO_THREAD_CHANNEL: int = 937840096021979236
 
 
 class AutoThread(commands.Cog):
-    def __init__(self, bot):
-        self.bot = bot
+    def __init__(self, bot: commands.Bot):
+        self.bot: commands.bot = bot
```

Comment:
Typecheckers can implicitly know the type if the parameter is typed, it's also a capital B but this isn't needed